files/cluster_provision.yml

```diff
@@ -51,49 +50,65 @@
                         set -ex
 
                         if [ ${machine_type} == "qdc" ]; then
-                                machine_list="qdcohpc, qdc01, qdc02, qdc03"
+                                machine_subarch=GrubWithOptionEfiboot
+                                machine_name="qdcohpc,qdc01,qdc02,qdc03"
                                 preseed_type="kickstart"
                                 kernel_desc="CentOS 7.5"
                                 initrd_desc="CentOS 7.5"
                                 preseed_name="CentOS Upstream"
-                                kernel_opts="ip=dhcp text inst.stage2=http://mirror.centos.org/altarch/7/os/aarch64/ inst.repo=http://mirror.centos.org/altarch/7/os/aarch64/ inst.ks=file:/ks.cfg"
+                                stagetwo="http://10.40.0.13/mirror.centos.org/altarch/7/os/aarch64/"
+                                repo="http://10.40.0.13/mirror.centos.org/altarch/7/os/aarch64/"
+                                qdc="earlycon console=ttyAMA0,115200"
+                                special=(${qdc} ${qdc} ${qdc} ${qdc})
```

Comment:
This fix did not work, nor did quoting. The parameters still separate by spaces, and qdc has a space inside, meaning the first machine gets `earlycon` and the second `console=..` and so on.